tests/unit-tests/test_dataloading.py

```diff
@@ -2667,24 +2667,22 @@ def test_realtime_infer_node_dataloader():
     src_idx_r1, dst_idx_r1 = g.edges(form='uv', etype=('n0', 'r1', 'n1'))
     all_src_idx = np.unique(np.concatenate([src_idx_r0, src_idx_r1]))
 
-    all_nodes = []
-    for input_nodes, seeds, blocks in dataloader:
-        assert 'n0' in input_nodes
-        assert input_nodes['n0'].shape[0] == len(all_src_idx)
-        assert 'n1' in input_nodes
-        
-        assert input_nodes['n1'].shape[0] == batch_size
-        assert 'n1' in seeds
-        assert len(blocks) == 1
-        all_nodes.append(seeds['n1'])
+    # should only has one mini batch
```

Comment:
You should also check the len of dataloader

Reply:
added.